### PR TITLE
fix: cassandra-tool typo and missing method

### PR DESCRIPTION
**Pull request by nicoloboschi**
# Description

Follow up of https://github.com/run-llama/llama_index/pull/13423 with some fixes

### CODE REVIEW

File: llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/cassandra_database_wrapper.py [hunk 1]
```diff
@@ -57,6 +56,18 @@ def run(
         else:
             raise ValueError("Fetch parameter must be either 'one', 'all', or 'cursor'")
 
+    def run_no_throw(
```
[the remark this answers is not in the document]
Comment (nicoloboschi) [reply llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/cassandra_database_wrapper.py:59]: yes it was referred here https://github.com/run-llama/llama_index/blob/80aa10e14c0a079225cd844b42ced9151dde0f37/llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/base.py#L38

[the remark this answers is not in the document]
Comment (nerdai) [reply llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/cassandra_database_wrapper.py:59]: ohhhh, got it! thanks 🙏🏼

**Review by nerdai — APPROVED** [review]
I'm a bit confused as to why these methods were missing in the first place?

**Comment by nicoloboschi** [comment]
> I'm a bit confused as to why these methods were missing in the first place?

I guess @pmcfadin did a last-minute refactor but didn't update the usages

**Pull request merged**